ui/admin/app/controllers/scopes/scope/aliases/index.js

```diff
@@ -73,11 +69,10 @@ export default class ScopesScopeAliasesIndexController extends Controller {
   @notifyError(({ message }) => message, { catch: true })
   @notifySuccess('notifications.clear-success')
   async clearAlias(alias) {
```

Comment:
I notice we have a route action `clearAlias` and `deleteAlias` as well as controller ones? Did we need multiple versions that do almost the same thing?

Reply:
yes we need as the other ones are for the aliases within target route

Reply:
We couldn't just share it? Since you can inject controllers?

Reply:
Can we share controllers from one route to another? 🤔

Reply:
I think technically yes but we can talk about this offline in tech time, this isn't blocking